packages/desktop/components/popups/AllotManaPopup.svelte

```diff
@@ -95,6 +95,7 @@
                 manaAllotments: { [accountId]: Number(rawAmount) },
             })
         } catch (err) {
+            error = err
```

Comment:
I think that the errors of `enough funds to generate mana` and `slots remaining...` should not be shown. wdyt?